mets.md

```diff
@@ -131,6 +137,39 @@ PROCESSOR := [A-Z0-9\-]{3,}
 `<mets:file ID="OCR-D-IMG_0001">`            | The unmanipulated source image
 `<mets:file ID="OCR-D-IMG-BIN_0001">`        | Black-and-White image
 
+#### Fulldownload
+
+The `ID` attribute prefix `FULLDOWNLOAD` followed by the file format (TEI, ALTO, hOCR, HTML, TXT, PDF) must be used for a requested file, their are the hole publication or digital book.
```

Comment:
No, this type of formulation requires that the documentation be read sequentially first.

What is important?
I think the documentation of the attribute value, with its specifics (prefix_format) + a characteristic description (whole book ...).

My proposal:

`For `mets:file` entries representative of the publication **as a whole** (not just a single page), the `ID` attribute MUST have  prefix `FULLDOWNLOAD_`, followed by the file format (`TEI`, `ALTO`, `hOCR`, `HTML`, `TXT`, `COCO`, `PDF`).`

Reply:
> No, this type of formulation requires that the documentation be read sequentially first.

Huh?

If you wanted to say that mentioning the structMap here already anticipates the respective section: of course it does, that's why it's also a hyperlink to that section! But you cannot make this fully linear if you also want to give a full example below. (And in your draft it is the next section which is really misordered unnecessarily.)

Reply:
I think you misunderstood something here.

>>No, this type of formulation requires that the documentation be read sequentially first. *referred to* Instead of the above `OCR-D-` prefix recommendation,`  That is also the reason why I would like this passage deleted.

Reply:
> I think you misunderstood something here.

It's possible. But you are making it hard for me to understand…

(Von mir aus auch gern auf Deutsch.)

> No, this type of formulation requires that the documentation be read sequentially first. _referred to_ Instead of the above `OCR-D-` prefix recommendation,` That is also the reason why I would like this passage deleted.

Can you please use Markdown properly so I can clearly see what you quote, what you add, what you emphasize and what you refer to verbatim? It's really hard to parse this kind of mixture and you already pointed out there must have been some confusion.

Reply:
Ok, so after direct consultation we identified the source of confusion. @tboenig is concerned that formulations like 
> Instead of the above `OCR-D-` prefix recommendation

(directed at text coherence) will require a linear reading flow, which is not appropriate for a specification. Instead, both subsections (the previous one about file ID naming for single pages and the new one on `FULLDOWNLOAD`) should be on the same level, and self-contained.

Reply:
```suggestion
For `mets:file` entries representative of the publication **as a whole**, the `ID` attribute MUST have  prefix `FULLDOWNLOAD_`, followed by the file format (`TEI`, `ALTO`, `hOCR`, `HTML`, `TXT`, `COCO`, `PDF`).

These entries SHOULD be referenced in the [structMap](#ocr-d-structmap) under `/mets:mets/mets:structMap[@TYPE="PHYSICAL"]/mets:fptr`.
```

Reply:
I cannot add suggestions for the previous subsection (because it is outside of the diff context), but roughly I would:
- introduce a level-3 heading beneath `File ID syntax` for either single-page files (with `OCR-D-...` convention) or whole-document files (with `FULLDOWNLOAD...` convention)
- add a note in the former that physical pages must be referenced in the physical structMap (also with a hyperlink)